R/render.r

```diff
@@ -422,7 +422,102 @@ data_deps <- function(pkg, depth) {
       )
     )
   )
-  deps <- bslib::bs_theme_dependencies(bs_theme)
+
+  # map secondary to component-active-bg (thus also dropdown-link-active-bg)
+  # unless a value was set by the user
+  if (is.null(pkg$meta[["template"]]$bslib$`component-active-bg`)) {
+    bs_theme <- bslib::bs_add_variables(
+      bs_theme,
+      "component-active-bg" = as.character(
+        bslib::bs_get_variables(bs_theme, "secondary")
+      )
+    )
+  }
+
+  # map body color to navbar colors
+  if (is.null(pkg$meta[["template"]]$bslib$`navbar-light-color`)) {
+    bs_theme <- bslib::bs_add_variables(
+      bs_theme,
+      "navbar-light-color" = "rgba($body-color, 0.8)",
```

Comment:
with the rgba function it seems I can't use the `as.character()` workaround that the assignment just before that (for `component-active-bg`) has.